app/models/post.rb

```diff
@@ -10,6 +16,11 @@ def recent_five
 
   private
 
+  def set_defaults
+    self.comments_counter = 0
+    self.likes_counter = 0
+  end
```

Comment:
[OPTIONAL] You don't need this here. You could have just set it in the migrations

Reply:
Yes @fobadara, I will make that change in the migrations. I simply wanted to try something new 😄